packages/mdc-floating-label/mdc-floating-label.scss

```diff
@@ -30,13 +30,11 @@
   transition: mdc-floating-label-transition(transform), mdc-floating-label-transition(color);
   cursor: text;
 
-  // stylelint-disable plugin/selector-bem-pattern
-  @include mdc-rtl(".mdc-text-field") {
+  @include mdc-rtl() {
```

Comment:
Nit: We normally omit empty parens when defining and including Sass mixins.

Our code base is somewhat inconsistent about that, of course, since we don't have a stylelint rule for it, but just FYI :slightly_smiling_face:

Reply:
ah gotcha - thanks :)